src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -205,32 +220,35 @@ public function getMarkingStore()
         return $this->markingStore;
     }
 
+    /**
+     * @param object     $subject
+     * @param Marking    $marking
+     * @param Transition $transition
+     *
+     * @return TransitionBlockerList
+     */
     private function doCan($subject, Marking $marking, Transition $transition)
```

Comment:
Typical exemple of useless PHPDoc ;) 
As this method is private, you can safely add a return type hint